src/components/Benchmarks/BenchmarkInfo.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Link from '@docusaurus/Link'
+import Translate from '@docusaurus/Translate'
+
+const BenchmarkInfo = (props) => {
+  return (
+    <div className="box">
+      <div className="level">
+        <div className="level-left">
+          <h2>{props.name}</h2>
+        </div>
+        <div className="level-right">
+          <Link to="https://github.com/fastify/fastify">
+            <span className="github-link">&nbsp;</span>
+          </Link>
+        </div>
+      </div>
+      <p>
+        {Math.round(props.req)}
+        <Translate> req/sec (</Translate>
+        <Link to={`https://github.com/fastify/benchmarks/blob/master/benchmarks/${props.name}.cjs`}>
+          <Translate>Check the code</Translate>
```

Comment:
Instead of this `Check code` link, we can just remove it an implement the framework name `<h2>{props.name}</h2>` as a link - little optimization, less code!